src/pages/TermsAndConditions.jsx

```diff
@@ -3,9 +3,19 @@ import useMobile from "../layout/Responsive";
 import { Helmet } from "react-helmet";
 import Header from "../layout/Header";
 import Footer from "../layout/Footer";
+import useCountryId from "../hooks/useCountryId";
 
 export default function TACPage() {
   const mobile = useMobile();
+
+  const countries = {
```

Comment:
Very minor, but to ensure consistency (and avoid redefining), could you instead use the `COUNTRY_NAMES` object from https://github.com/PolicyEngine/policyengine-app/blob/master/src/data/countries.js?